subsys/bluetooth/common/dummy.c

```diff
@@ -4,11 +4,13 @@
  */
 
 /*
- * Copyright (c) 2017 Nordic Semiconductor ASA
+ * Copyright (c) 2024 Nordic Semiconductor ASA
```

Comment:
nit: don't think we update the copyright years

Reply:
Should I leave it at 2017? Or put '2017-2024'?

Reply:
There may be differences on the kind of legal guidance different companies give, but in my experience the more common practice is to have ranges. I guess you could think of it as saying that depending on the line that you're looking at in the file its copyright year may vary within a given range. Also, copyright can expire, so it feels unintuitive to me that the expiry for all content in the file, no matter how much new content was added many years after the file was initially created, would still be anchored on the creation year of the file.